src/main/docker/Dockerfile

```diff
@@ -20,12 +20,11 @@ RUN zypper -n install python-devel && \
     zypper -n install python-pip && \
     zypper -n install python-matplotlib && \
     zypper -n install zlib-devel && \
-    zypper -n install python-numpy && \
+    zypper -n install python-numpy-devel && \
     zypper -n install python-lxml && \
     zypper -n install python-scipy && \
     zypper -n install gcc-c++ && \
     pip install regex==2014.12.24 && \
-    pip install numpy==1.16.1 && \
```

Comment:
FYI...I removed this line because I see this in the build log:
`pip install numpy==1.16.1
Requirement already satisfied: numpy==1.16.1 in /usr/lib64/python2.7/site-packages (1.16.1)`